dll/win32/windowscodecs/typeof.h

```diff
@@ -80,7 +84,11 @@ typedef void (__cdecl typeof(png_set_PLTE))(struct png_struct_def *, struct png_
 typedef void (__cdecl typeof(png_set_tRNS))(struct png_struct_def *, struct png_info_def *, const unsigned char *, int, const struct png_color_16_struct *);
 typedef void (__cdecl typeof(png_set_filter))(struct png_struct_def *, int, int);
 typedef void *thandle_t_1;
+#ifdef _WIN64
+typedef size_t (*TIFFReadWriteProc_1)(thandle_t_1, void *, __typeof_intptr);
```

Comment:
This should return a signed value. The definition resolves to `signed __int64` for x64

Reply:
Fixed